Mathlib/MeasureTheory/Integral/Periodic.lean

```diff
@@ -221,14 +225,56 @@ protected theorem intervalIntegral_preimage (t : ℝ) (f : UnitAddCircle → E)
 
 end UnitAddCircle
 
-variable {E : Type*} [NormedAddCommGroup E] [NormedSpace ℝ E]
-
+/-!
+## Interval integrability of periodic functions
+-/
 namespace Function
 
 namespace Periodic
 
+variable {E : Type*} [NormedAddCommGroup E]
+
 variable {f : ℝ → E} {T : ℝ}
 
+/-- A periodic function is interval integrable over every interval if it is interval integrable
+over one period. -/
+theorem intervalIntegrable {t : ℝ} (h₁f : Function.Periodic f T) (hT : 0 < T)
+  (h₂f : IntervalIntegrable f MeasureTheory.volume t (t + T)) (a₁ a₂ : ℝ) :
+    IntervalIntegrable f MeasureTheory.volume a₁ a₂ := by
+  -- Replace [a₁, a₂] by [t - n₁ * T, t + n₂ * T], where n₁ and n₂ are integers
```

Comment:
Your comments and code disagree: do you use natural numbers or integers? (I think "whichever is more convenient for this proof" is a good-enough answers, and am not sure if that's naturals or integers. Presumably, both get coerced to real numbers anyway.)

Reply:
Corrected the comment, speaking about "natural numbers" now.